src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -294,20 +294,20 @@ public NetworkPeer(IDateTimeProvider dateTimeProvider, ILoggerFactory loggerFact
             this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
 
             this.Behaviors = new NetworkPeerBehaviorsCollection(this);
-            this.PeerAddress = new NetworkAddress();
+            this.PeerAddress = new IPEndPoint(IPAddress.Loopback, 1);
```

Comment:
Is this initialization correct? Can't we just have it null to start with?

Reply:
This constructor will be removed, but it is used now I think some tests would crash or fail if this was null.But I am not sure about that, but as it will be deleted, it probably doesn't need to be investigated further.

Reply:
OK all good then!